lerna.json

```diff
@@ -5,8 +5,8 @@
   "version": "independent",
   "command": {
     "publish": {
-      "allowBranch": "master",
-      "message": "chore(release): publish :tada:",
+      "allowBranch": "automation/lerna",
```

Comment:
The `allowBranch` field dictates which branches `lerna version` and/or `lerna publish` can run on. As Lerna is run after a PR is merged into `master`, `lerna version` still needs to be allowed to run on `master`. It doesn't actually make sense to have this be aligned to the generated PR branch as we aren't running `lerna version` in the CI for the PR itself (only once its merged). That said, there is the open question of how/when the Git tags and Github release would be created/pushed based on the Lerna-generated PR, that might dictate needing to run `lerna version` in some form from a PR itself.

Related, the "Publish from package.json" GH workflow (the manual step after you merge a PR) is running `lerna publish from-package ...`. As such, the `allowBranch` still needs to be set to `master` at least for the "publish" command, too.

If needed, `allowBranch` does accept a list for multiple branches as well (e.g., if we do need to run `lerna version` on the generated branch).

Reply:
I didn't read the lerna docs and made an assumption. Will go digest this now.